cdap-system-app-api/src/main/java/io/cdap/cdap/api/service/operation/LongRunningOperation.java

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
We already have a class called `Operation` in lineage hence the name `LongRunningOperation`